src/period.cpp

```diff
@@ -8,15 +8,14 @@
 #include "nanotime/pseudovector.hpp"
 #include "nanotime/utilities.hpp"
 
-// See comment in <R_Ext/Complex.h> in R>=4.3.0. As of 2024-10-30
-//   clang trunk with -Wmissing-braces requires the double-braced
-//   approach for creating Rcomplex objects.
+// See comment in <R_Ext/Complex.h> in R>=4.3.0 and #134. Old
+//   approach to do `Rcomplex{ re, im }` now gives compiler issue
+//   on some compilers, R versions.
 inline Rcomplex makeComplex(double *re, double *im) {
```

Comment:
Wait, I am just eye-balling this after dinner but this would required all calling instances to pass by (icky, to a C++ programmer) pointer.  Really?

Reply:
NB this is stale